CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,36 @@
+# Code of Conduct {#about_code_of_conduct}
```

Comment:
>  If it does, it would be nice since then also the pure MD would render properly.

But even then the link wouldn't work in pure MD. I had to choose one of the two, and I thought that doxygen would be more important as everything is laredy "top-level" in the repository and people will easily find it.

Reply:
ok